rasa/core/nlg/response.py

```diff
@@ -75,12 +75,20 @@ def _responses_for_utter_action(
         )
 
         if conditional_channel:
+            logger.debug(
+                "Selecting response variation with condition: "
+                f"'{conditional_channel[0].get('condition')}'"
```

Comment:
To double-check on your intention here - you want to log the condition data, i.e. matched slot name and value?
If that's so perhaps we should format this a bit nicer to the user, otherwise it will come out as a list object with a dict, so for example extracting slot name and value from condition and using these in the f-string.

I would also use the constant `RESPONSE_CONDITION` for consistency.

Reply:
Yeah, it'd be for logging slot names and values. Make sense, will update